src/Symfony/Component/AssetMapper/ImportMap/RemotePackageDownloader.php

```
@@ -69,11 +72,15 @@ public function downloadPackages(?callable $progressCallback = null): array
             return [];
         }
 
-        $contents = $this->packageResolver->downloadPackages($remoteEntriesToDownload, $progressCallback);
+        try {
+            $contents = $this->packageResolver->downloadPackages($remoteEntriesToDownload, $progressCallback);
+        } catch (TimeoutException|TransportException $exception) {
+            throw new LogicException($exception->getMessage());
+        }
```

I don't think this is a good idea, as it's hiding the real exception. Also, this is not a "Logic" exception.

---

Ok, I understand but how would y'all like to see this in that case. It's hard to guess what direction I need to go with this.

Again I'm new to the Symfony world and just want to fix this specific issue because it is breaking deployments. And while I want to fix it I do want to do it while communicating.

Let me know if you have any thoughts on how to improve things. Have a nice evening!